deploy/crds/shipwright.io_builds.yaml

```diff
@@ -165,6 +165,18 @@ spec:
               source:
                 description: Source refers to the Git repository containing the source code to be built.
                 properties:
+                  container:
```

Comment:
Is the `source.url` field still required ?

Reply:
No, the `source` `url` should be optional. I will change that.